src/bb-library/Registrar/Adapter/srsx.php

```diff
@@ -10,13 +10,16 @@
  * https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE
  * If you did not receive a copy of the license and are unable to
  * obtain it through the world-wide-web, please send an email
- * to license@boxbilling.org so we can send you a copy immediately.
+ * to license@fossbilling.org so we can send you a copy immediately.
```

Comment:
Is the email address working? 

```
De reactie van de externe server was:
550 5.1.1 Address does not exist. zhrkxOKRhJ9y

```

Or we should fix the email or remove this line. License is also attached in the project files

Reply:
That's a good question, but since the changes in the commit that was merged earlier used it (and a few other mail addresses that I had never seen before), I assumed there was a catch-all address working. 

Where do emails to FOSSBilling go actually, who has access to receive them?

Reply:
I don't think it is currently setup:
<img width="857" alt="Screenshot 2022-09-30 at 10 33 55" src="https://user-images.githubusercontent.com/9754650/193228452-b96a0270-5be2-4ae1-bbc5-6c4e7ced5607.png">

Reply:
Removed. it was unnecessary anyway - the license is easy enough to find :)